x/dex/keeper/contract.go

```diff
@@ -50,6 +50,18 @@ func (k Keeper) GetContract(ctx sdk.Context, contractAddr string) (types.Contrac
 	return res, nil
 }
 
+func (k Keeper) GetContractGasLimit(ctx sdk.Context, contractAddr []byte) (uint64, error) {
```

Comment:
why is this `[]byte` instead of `AccAddress`? Then we can know its already validated and simply call `.String()` on it, right?